index.html

```diff
@@ -4,26 +4,146 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="style.css">
+  <script src="loadteams.js" defer></script>
   <script src="actions.js" defer></script>
-  <title>FICA Football Tournament</title>
+  <title>FICA Soccer Tournament</title>
 </head>
 <body>
+  <nav id="mobile-menu">
+    <img id="close-menu" src="img/close_mobile_menu.png" width=20 height=19 alt="Close">
+    <div id="mobile-menu-ul">
+      <hr class="hr1">
+      <p class="list-item"><a class="asnormal" href="about.html">About</a></p>
+      <hr class="hr1">
+      <p class="list-item"><a class="asnormal" href="#">Tournament</a></p>
+      <hr class="hr1">
+      <p class="list-item"><a class="asnormal" href="#">Accreditation</a></p>
+      <hr class="hr1">
+      <p class="list-item"><a class="asnormal" href="#">Sponsors</a></p>
+      <hr class="hr1">
+      <p class="list-item"><a class="asnormal" href="#">News</a></p>
+      <hr class="hr1">
+    </div>
+  </nav>
   <header>
-    <nav>
-
-    </nav>
+    <div id="mobile-header">
+      <img id="hamburger-menu" src="img/mobile_menu.png" width=20 height=18 alt="Menu">
+      <img src="img/logo.png" width=130 height=51 alt="FICA Tournament">
+    </div>
+    <div id="desktop-header">
+      <nav id="desktop-nav">
+        <img src="img/twitter.png" width=20 height=17 alt="Twitter">
+        <img src="img/facebook.png" width=9 height=18 alt="Facebook">
+        <a href="#" class="asnormal calcright">Spanish</a>
+      </nav>
```

Comment:
[OPTIONAL] - This is marked as optional because the previous reviewer did not mention it.

The Twitter icon on the top navigation doesn't display properly.
![image](https://user-images.githubusercontent.com/38088818/163624743-95413db4-4f35-42c4-b0c9-02140e3f5bf5.png)

Reply:
Corrected. I don't know why the Twitter.png image ended up capitalized in Github when in my local folder it's lowercase.

The acknowledgment was under the License title, I extended it by putting it also under Acknowledgements.